_config.yml

```diff
@@ -0,0 +1,4 @@
+collections:
+  services:
+    output: true
```

Comment:
Set `output: false` to avoid generating pages for each service

Reply:
Fixed